openhands/server/session/agent_session.py

```diff
@@ -268,10 +286,18 @@ def _create_controller(
                 'Runtime must be initialized before the agent controller'
             )
 
+        # Determine port mapping instructions based on Runtime type
+        port_instructions = ''
+        if isinstance(self.runtime, EventStreamRuntime):
+            port_instructions = 'Use ports 4141 and 4142 when serving applications'
+        elif isinstance(self.runtime, RemoteRuntime):
+            port_instructions = 'Use ports 81623 and 81624 when serving applications'
```

Comment:
let's remove this part--don't love that it's hard-coded, and it's just a debug log

Reply:
The code you're referring to aren't the debug statements. I removed them below since I assume that is what you meant